code/script.js

```diff
@@ -35,17 +42,52 @@ const showMessage = (message, sender) => {
 
 // Starts here
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
 
+  showMessage("Welcome to report error messages about our municipal streets! Do you want to make a report?", 'bot');
+};
+
+
+
+  
+  // Display the user's answer in the chat
+  // Here you can process the user's answer, ask the next question, etc.
+  // For now, let's just display a response from the bot:
+
+
+
+
+
+const handleUserAnswer = event => {
+  event.preventDefault ();
+  const userAnswer = userInput.value.trim().toLowerCase();
+  showMessage(userAnswer, 'user');
+  setTimeout(() => {
+
+  if (userAnswer === 'yes') {
+    showMessage("Svar", 'bot');
+    
+  } else if (userAnswer === 'no') {
+    showMessage("OK!", 'bot');
+
+  } else { showMessage("I'm sorry, I didn't understand your response. Please answer with 'yes' or 'no'.", 'bot');
+  }
+  reply();
```

Comment:
A suggestion for the next step is to define each function where you put in not only the bot reply to user's previous answer "thank you..." but also the next question you would like it to ask. It could include the following parts: 
1. Bot reply to user's previous answer 
2. Bot ask the next question (can be put together with part 1)
3. Add the intended question and its alternatives in innerHTML, for example buttons, dropdown list etc
4. Store the user's choice in variable(s)
5. Add event listener to user's choice of alternative(for button, we often listen to event "click"; for dropdown list, it's the event "change" --> something happens)
6. Remove the buttons/dropdown list etc 
7. Invoke the next question function with a delay to move on to the next one